config-templates/tables.postgres.sql

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE "statistics_sums" (
+  "id" bigint GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  "year" integer NOT NULL,
+  "month" integer DEFAULT NULL,
+  "day" integer DEFAULT NULL,
+  "idp_id" integer NOT NULL DEFAULT 0,
+  "sp_id" integer NOT NULL DEFAULT 0,
+  "logins" integer DEFAULT NULL,
+  "users" integer DEFAULT NULL,
+  CONSTRAINT "year" UNIQUE ("year","month","day","idp_id","sp_id")
+);
+
+CREATE INDEX "statistics_sums_idp_id" ON "statistics_sums" ("idp_id");
+CREATE INDEX "statistics_sums_sp_id" ON "statistics_sums" ("sp_id");
+
+CREATE TABLE "statistics_per_user" (
+  "day" date NOT NULL,
+  "idp_id" integer NOT NULL,
+  "sp_id" integer NOT NULL,
+  "user" character varying(255) NOT NULL,
```

Comment:
Maybe use varchar(255) instead? Probably just a cosmetic thing, but I think VARCHAR is more common.